### PR TITLE
fix for pause in sms response

### PR DESCRIPTION
**Pull request by pjaudiomv**
#68
this should fix this and still have pauses where we want them

### CODE REVIEW

File: meeting-search.php [hunk 1]
```diff
@@ -65,11 +65,12 @@
         if ($results_counter == $results_count) break;
     }
 
-    echo "<Pause length=\"2\"/>";
+    
 
     // Do not handle for the SMS gateway
     if (!isset($_REQUEST["SmsSid"]) && count($filtered_list) > 0) {
-        if (count($sms_messages) > 0) { ?>
+        if (count($sms_messages) > 0) {
```
Comment (radius314) [inline meeting-search.php]: can you just move to have it below the first `if`?

Comment (radius314) [reply meeting-search.php]: only need one instance of this.

Comment (pjaudiomv) [reply meeting-search.php]: yes that makes much more sense

File: meeting-search.php [hunk 2]
```diff
@@ -65,11 +65,12 @@
         if ($results_counter == $results_count) break;
     }
 
-    echo "<Pause length=\"2\"/>";
+
```
Comment (radius314) [inline meeting-search.php]: remove extra space

File: meeting-search.php [hunk 3]
```diff
@@ -65,11 +65,12 @@
...
     }
 
-    echo "<Pause length=\"2\"/>";
+    
 
     // Do not handle for the SMS gateway
     if (!isset($_REQUEST["SmsSid"]) && count($filtered_list) > 0) {
-        if (count($sms_messages) > 0) { ?>
+      echo "<Pause length=\"2\"/>";
+        if (count($sms_messages) > 0) {  ?>
```
Comment (radius314) [inline meeting-search.php]: remove extra space after `{`

File: meeting-search.php [hunk 4]
```diff
@@ -82,7 +83,7 @@
             <Gather numDigits="1" timeout="10"
                     action="post-call-action.php?Payload=<?php echo urlencode( json_encode( $sms_messages ) ) ?>"
                     method="GET"/>
-        <?php } elseif (isset($GLOBALS['infinite_searching']) && $GLOBALS['infinite_searching']) { ?>
+        <?php } elseif (isset($GLOBALS['infinite_searching']) && $GLOBALS['infinite_searching']) {  ?>
```
Comment (radius314) [inline meeting-search.php]: remove extra space after `{`

File: meeting-search.php [hunk 5]
```diff
@@ -65,11 +65,12 @@
         if ($results_counter == $results_count) break;
     }
 
-    echo "<Pause length=\"2\"/>";
+    
 
     // Do not handle for the SMS gateway
     if (!isset($_REQUEST["SmsSid"]) && count($filtered_list) > 0) {
-        if (count($sms_messages) > 0) { ?>
+      echo "<Pause length=\"2\"/>";
```
Comment (radius314) [inline meeting-search.php]: make sure `echo` is aligned with `if` on the line below.

**Comment by radius314** [comment]
there are 2 other changes to make

**Comment by radius314** [comment]
please look at all the comments https://github.com/radius314/yap/pull/69/files

**Comment by pjaudiomv** [comment]
sorry it got confusing, i should of done them in order, would of made a lot more sense

**Comment by radius314** [comment]
you can do all of them at once, they don't need to be individual commits.

**Pull request merged**